src/main/java/org/opentripplanner/framework/tostring/ToStringBuilder.java

```diff
@@ -95,6 +95,10 @@ public ToStringBuilder addBool(String name, Boolean value) {
     return addIfNotNull(name, value);
   }
 
+  public ToStringBuilder addBool(String name, Boolean value, Boolean ignoreValue) {
+    return addIfNotIgnored(name, value, ignoreValue, Object::toString);
+  }
+
```

Comment:
I prefer the `addBoolIfTrue` over this. For example `useAvailabilityInformation=false` is not as readable as `skipAvailabilityInfo`. We try to keep toStrings short and readable. You would need to repeat the default value in the toString inside the class to do this, but I prefer that for the extra readability.

Consider adding JavaDoc recommending the `addBoolIfTrue` or converting the usage of this. Note! That the `addBoolIfTrue` fits well with things that are off by default, and not so good with things that are on. Note also that the consequences of flipping the value in the toString is very small - it is just confusing.

Reply:
This is not important!

Reply:
I ended up renaming the parameter `ignoreRealtimeAvailability`. I was considering using the `excludeRealtimeUpdates` naming we used the plan pr but this paramete doesn't really exclude all realtime updates as the parks itself are added through realtime updates, it just ignores the realtime availability. I only did the renaming in the preferences, the REST API parameter stays as is.